docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -81,7 +81,9 @@ The following overload actions are supported:
     - Envoy will immediately respond with a 503 response code to new requests
 
   * - envoy.overload_actions.disable_http_keepalive
-    - Envoy will stop accepting streams on incoming HTTP connections
+    - Envoy will drain HTTP/2, HTTP/3 connections using `GOAWAY` with a drain
```

Comment:
hi @KBaichoo can you use double backticks for rst please ` `` ` - single backtick creates italics, whereas we want an inline literal - you may want to consider likewise for some of the other literals - eg `HTTP/2` etc but maybe not

Reply:
Whoops, thanks for catching that. GOAWAY is now inline literal